aws/resource_aws_lambda_function.go

```diff
@@ -426,8 +426,8 @@ func resourceAwsLambdaFunctionCreate(d *schema.ResourceData, meta interface{}) e
 		params.Tags = keyvaluetags.New(v.(map[string]interface{})).IgnoreAws().LambdaTags()
 	}
 
-	// IAM changes can take 1 minute to propagate in AWS
-	err := resource.Retry(1*time.Minute, func() *resource.RetryError {
+	// IAM changes can take some time to propagate in AWS
+	err := resource.Retry(2*time.Minute, func() *resource.RetryError {
```

Comment:
Hi @camlow325 👋  Thanks for submitting this and its a good change! It might be nice to switch this to the shared constant we have available nowadays, so all resources across the provider are consistent. You can see an example of this here: https://github.com/terraform-providers/terraform-provider-aws/pull/14263/files

```suggestion
	err := resource.Retry(iamwaiter.PropagationTimeout, func() *resource.RetryError {
```

And its associated import: `iamwaiter "github.com/terraform-providers/terraform-provider-aws/aws/internal/service/iam/waiter"` -- thanks again!

Reply:
Thanks for the review, @bflad. Your suggestion makes a lot of sense to me. I went ahead and amended it into a new commit, 95f8b09.